zerver/openapi/zulip.yaml

```diff
@@ -24758,6 +24805,22 @@ components:
             change, the permission to administer channels was limited to realm
             administrators.
 
+            [setting-values]: /api/group-setting-values
+    CanSendMessageGroup:
+      allOf:
+        - $ref: "#/components/schemas/GroupSettingValue"
+        - description: |
+            A [group-setting value][setting-values] defining the set of users
+            who have permission to post in this channel.
+
+            Note that a user who is a member of the specified user group must
+            also [have access](/help/channel-permissions) to the channel in
+            order to post in the channel.
+
+            **Changes**: New in Zulip 10.0 (feature level 333). Previously
+            `stream_post_policy` field used to control the permission to
+            post in the channel.
```

Comment:
We usually give the mapping of how `stream_post_policy` works; is that left off here because we're keeping a deprecated version of it in some places? Maybe reasonable.

Reply:
Yes, we do not describe the mapping for other similar settings as well.